recipe/install-libgcc-no-gomp.sh

```diff
@@ -0,0 +1,9 @@
+source ${RECIPE_DIR}/install-libgcc.sh
+
+# remove and relink things
+rm -f ${PREFIX}/lib/libgomp.so
+rm -f ${PREFIX}/${CHOST}/sysroot/lib/libgomp.so
+
+# make the right links
+ln -s ${PREFIX}/lib/libgomp.so.${libgomp_ver:0:1} ${PREFIX}/lib/libgomp.so
```

Comment:
```suggestion
ln -s ${PREFIX}/lib/libgomp.so.${libgomp_ver} ${PREFIX}/lib/libgomp.so
```

Reply:
This suggestion is not right. I thought we wanted libgomp to point at libgomp.so.1. We then point libgomp.so.1 at either libgomp.so.1.0.0 or libgomp.so. Here libgomp_ver is 1.0.0 so you have only keep the first character.

Reply:
Nope, we want `libgomp.so` to point at `libgomp.so.1.0.0` so that the SONAME will always be `libgomp.so.1`. Executable and other shared libraries will load `libgomp.so.1` which is going to swapped with llvm one.

Reply:
Ohhhhhh. I am so sorry! I must have misunderstood you. Thank you so much! Everyday I learn something new! :)